src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/courtsentencing/Transformer.kt

```diff
@@ -47,7 +48,13 @@ fun CourtCaseResponse.toMigrationDpsCourtCase() = MigrationCreateCourtCase(
     },
   ),
   active = this.caseStatus.code == "A",
-
```

Comment:
DPS has 3 states that represent:

- case/charge not involved in any case linking
- is involved and is the source; so is read-only therefore merged = true
- is involved and is the target; so not read-only  therefore merged = false